src/components/Card/index.js

```diff
@@ -28,7 +28,7 @@ export default React.memo(function Card(props) {
       ref={ref}
       style={{ fontSize }}
       data-testid='card'
-      id={props.id}
+      id={props.id + '_' + props.idx}
```

Comment:
I think the following would be better so we only have the `id` when no `idx` is provided:

```js
id={[props.id, props.idx].join('_')}
```

This way other parts of the site like the battle sim would keep working.